spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 require 'capybara/rspec'
 require 'simplecov'
 require 'simplecov-console'
+require '/Users/mohammedmiah/rps-challenge-apprenticeships/spec/features/web_helpers.rb'
```

Comment:
Don't use the full path. you can just use:

require 'features/web_helpers.rb'

If someone was to pull your code, they won't have /Users/mohammedmiah directories on there machine.